db/crud.go

```diff
@@ -56,6 +56,29 @@ func (db *DatabaseContext) GetDoc(docid string) (*document, error) {
 	return doc, nil
 }
 
+func (db *DatabaseContext) GetDocSyncData(docid string) (syncData, error) {
```

Comment:
I just (force) pushed up an amended commit with a TODO and a comment on that function